### PR TITLE
Fix empty state cases for remove wallets and groups

### PR DESCRIPTION
**Pull request by zerts**

### CODE REVIEW

File: src/ui/pages/ManageWallets/WalletAccount/WalletAccount.tsx [hunk 1]
```diff
@@ -301,6 +302,11 @@ export function WalletAccount() {
               },
             ]}
           />
+          {!removeAddressMutation.isLoading && removeAddressMutation.error ? (
```
Comment (everdimension) [inline src/ui/pages/ManageWallets/WalletAccount/WalletAccount.tsx:305]: Can this be changed to
```suggestion
          {removeAddressMutation.isError ? (
```

**Review by everdimension — APPROVED** [review]

**Comment by everdimension** [comment]
Let's QA this one thoroughly pls

**Comment by everdimension** [comment]
Wrong button :)

File: src/ui/App/App.tsx [hunk 2]
```diff
@@ -193,6 +181,7 @@ function Views({ initialRoute }: { initialRoute?: string }) {
             element={
               <SomeKindOfResolver
                 noUser={<Navigate to="/intro" replace={true} />}
+                noWallet={<Navigate to="/get-started?intro" replace={true} />}
```
Comment (vyorkin) [inline src/ui/App/App.tsx:184]: 👍

File: src/ui/pages/ManageWallets/ManageWallets.tsx [hunk 3]
```diff
@@ -184,6 +184,10 @@ function WalletGroups() {
     return null;
   }
 
+  if (!walletGroups?.length) {
+    return <Navigate to="/get-started?intro" replace={true} />;
+  }
```
Comment (vyorkin) [inline src/ui/pages/ManageWallets/ManageWallets.tsx:189]: 👍

**Review by vyorkin — APPROVED** [review]

**Pull request merged**